schemas/v3.1/meta/base.schema.yaml

```diff
@@ -1,7 +1,7 @@
-$id: https://spec.openapis.org/oas/3.1/meta/base
+$id: https://spec.openapis.org/oas/3.1/meta/base/WORK-IN-PROGRESS
```

Comment:
We currently don't publish or reference date-stamped iterations for this file. 

Do we want to change this?

Reply:
@ralfhandl yes, because it never should have been this way in the first place.